pkg/containerutil/containerutil.go

```diff
@@ -272,7 +272,11 @@ func Start(ctx context.Context, container containerd.Container, flagA bool, clie
 		}
 	}
 	detachC := make(chan struct{})
-	task, err := taskutil.NewTask(ctx, client, container, flagA, false, flagT, true, con, logURI, detachKeys, namespace, detachC)
+	flagAStreams := []string{}
+	if flagA {
+		flagAStreams = []string{"STDOUT", "STDERR"}
```

Comment:
Just want to double check these are the streams we want to forward? Based on the [function description](https://github.com/containerd/nerdctl/blob/947932605f09af80a96db1ad9fde91cf67a7a210/pkg/containerutil/containerutil.go#L212) I assumed you'd be forwarding stdin + stdout but I'm not certain.

Reply:
According to the nerdctl documentation, `start -a` should only attach `STDOUT` and `STDERR`. I think this may be a typo in the function description. 

> -a, --attach: Attach STDOUT/STDERR and forward signals [(from nerdctl start command reference)](https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md#whale-nerdctl-start)